api/api/wsgi.py

```diff
@@ -12,7 +12,7 @@
 from django.core.wsgi import get_wsgi_application
 
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'api.settings')
-sys.path.append('/var/simra/SimRa2/api')
-sys.path.append('/var/simra/SimRa2/api/api')
+sys.path.append('/home/sfuehr/Documents/TUB-WI/S7_BA_SimRa/simra-visualizations-server/api')
```

Comment:
Sollten wir so nicht pushen. Entweder bei /var/simra/SimRa2 bleiben, oder das ganze in environment variables auslagern.

Reply:
Warum funktioniert hier ein relativer Pfad nicht? Oder kann man sich in Python nicht auch à la „realpath .“ oder pwd den Pfad holen?